src/components/Blockquote/Blockquote.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import { childrenPropType } from '../../util/shared-prop-types';
+import Text from '../Text';
+
+const baseStyles = ({ theme }) => css`
+  label: blockquote;
+  padding-left: ${theme.spacings.kilo};
+  border-left: 2px solid ${theme.colors.p500};
+`;
+
+const gigaStyles = ({ theme, size }) =>
+  size === Text.GIGA &&
+  css`
+    label: blockquote--giga;
+    padding-left: ${theme.spacings.mega};
+    border-left: 3px solid ${theme.colors.p500};
+  `;
+
+const StyledText = styled(Text)`
+  ${baseStyles};
+  ${gigaStyles};
+`;
+
+/**
+ * Indented and italicised text to denote a quotation.
+ */
+const Blockquote = ({ children, ...props }) => (
```

Comment:
Hey, I know I'm just hoping in, but shouldn't we wrap the StyledText into a blockquote instead of changing the Text element to be rendered as one?

Reply:
As we discussed offline, our perception of the Text component differs — in my opinion, this is exactly its purpose: a generic text component that can be used with any element and provides consistent font styles.